flake-modules/dev/default.nix

```diff
@@ -52,6 +52,7 @@
             "**.scm"
             "**.svg"
             "**/man/*.5"
+            "/docs/gfm-alerts-to-admonitions/tests/**/*.yml"
```

Comment:
Any reason why? (we have a yaml formatter configured IIRC)

Reply:
`pytest-regression` performs a `diff`, so the formatter messes with it

Reply:
Makes sense. Could you note that in a comment?